.travis.yml

```diff
@@ -3,22 +3,25 @@ sudo: false
 language: c
 
 env:
-  - GHCVER=7.8.4
-  - GHCVER=7.10.2
+  - GHCVER=7.8.4 CABALVER=1.22
```

Comment:
Let's leave discussing `tinc` vs `new-build` for a different PR then. 

I'm okay either with 1.22 or 1.24 - on the one hand, having different ones maybe helps check that everything works with both; on the other, it's a little cleaner with a single one.